wpeverest/user-registration
# Fix - Conditional logic not working in WooCommerce checkout page

### All Submissions:






### Changes proposed in this Pull Request:
Fixes - Conditional logic not working on the WooCommerce checkout page.

Depends on https://github.com/wpeverest/user-registration-woocommerce/pull/13

### How to test the changes in this Pull Request:

1. Setup WooCommerce and setup some Conditional Logic rules
2. Go to the checkout page
3. See if the conditional logic is behaving properly

### Types of changes:




### Other information:




### Changelog entry

> Fix - Conditional logic not working in WooCommerce checkout page
